### DISCUSSION TITLE
Web3modal CDN example does not work

### DISCUSSION THREAD

**Discussion by achille1017** [WalletConnect/walletconnect-monorepo#3450] [Developer Support] [answered]
https://github.com/WalletConnect/web3modal-examples/tree/main/web3modal-wagmi-html-cdn

When trying to run it from a flask server I got this error in my browser console : 
Uncaught (in promise) ReferenceError: process is not defined
    <anonymous> hey-listen.es.js:3
[hey-listen.es.js:3:4](https://unpkg.com/node_modules/hey-listen/dist/hey-listen.es.js)
    initUi client.ts:48
    AsyncFunctionThrow self-hosted:856
    (Asynchrone : async)
    He client.ts:25
    <anonyme> main.js:38
    
    
The line triggering the error is 
  export const web3Modal = new Web3Modal(
    {
      projectId,
      walletImages: {
        safe: "https://pbs.twimg.com/profile_images/1566773491764023297/IvmCdGnM_400x400.jpg",
      },
    },
    ethereumClient
  );

**Comment by glitch-txs** [comment, marked as answer]
We removed the polyfills from the library, you can add them manually:
https://github.com/WalletConnect/web3modal/issues/1192

**Comment by dappbuildernet** [comment]
Still, with all the above mentioned real CDN version is not working.

The reason for need in CDN is that users can get pre-built HTML+JS code that can be easily uploaded to their websites, with web3modal connection available there. There is no way to do this with NPM/React or any other way. Only pure HTML + JS.
Your current example is not working - QR code is not loading (though it is working in v2), coinbase wallet is not loading + other console errors.
Please kindly check the CDN version - users really need it. Thank you.

**Reply by mortzaCFT** [reply]
You can use this CDN's its seems working for me:
<script src="https://unpkg.com/web3@latest/dist/web3.min.js"></script>
<script src="https://unpkg.com/@walletconnect/web3-provider"></script>

Also here is the official doc:
https://docs.walletconnect.com/advanced/walletconnectmodal/about



But i do have another error using this:

<script type="module">
  const modal = new WalletConnectModal ({
    projectId: '89ff2f66cafc43d6b57b6ac685fbc965',
    chains: ['eip155:1'],
  });
 </script>
 
 The error says:
 Uncaught (in promise) ReferenceError: process is not defined
    at hey-listen.es.js:3:5
    
    
As being says , this error Uncaught (in promise) ReferenceError: process is not defined is occurring because the process object is not available in the browser environment.


So as we know the process object is a Node.js global object that provides information about the current process, such as the current working directory, environment variables, and more. It is not available in the browser environment, where JavaScript code is executed by the browser's JavaScript engine,..

But the official doc says we should use this lib :  @walletconnect/modal
 but this OFFICIAL LIB trying to use the process object, which is causing the ERROR.
 
 How to reslove?
 I dont know :/
